README.md

```diff
@@ -4,379 +4,7 @@ This SDK supports AndroidStudio.
-- Improved behaviour of the reconnect screen if connection is lost
-- Upgraded Tokbox to version 2.9.x
-- Use Tokbox directly from maven repository, it has now been removed as static aar
-
-Bugs:
-- Fixed race condition which could result in broken image uploads
-
-Migration Guide:
-- Add the following entry to the repositories in your build.gradle:
-```
-maven { url  "http://tokbox.bintray.com/maven" }
-```
-- Add the following entry to your dependencies:
-```
-compile 'com.opentok.android:opentok-android-sdk:2.9.+'
-```
-- Update reference to IDnow SDK to 2.8.0 in your dependencies
-
-### 2.7.0
-
-Improvements
-- Switched to OkHttp for websockets which increased speed and stability
-- Updated proguard rules for OkHttp
-- Gson updated to 2.4
-
-### 2.6.1
-
-Bugfixes
-- Fixed a rare crash when initializing the camera using the IDnow videoserver
-
-### 2.6.0
-
-Bugfixes
-- Correctly applied custom colors also for certain popups
-- Improved connection handling of the initial websocket
-- Fixed crash related to VP8 encoding
-- Fixed performance problems when performing severals idents in a row
-- Better echo cancellation
-
-### 2.5.0
-
-Improvements
-- Added support for spatial and temporal resampling of the video stream
-- Better handling slow STUN requests
-- Prevent parallel uploading of images for faster uploads
-- Return success if user aborts on the goodbye screen instead of failure
-- Sample mobile number is now "e.g. 0176123..."
-
-Bugfixes
-- Fixed crash on some devices when switch from back to front camera
-- Fixed bug where audio was not reconnected if user temporarily lost connection
-
-### 2.4.0
-
-Improvements:
-- Added jailbreak / rooting detection
-- Better wording for Ident-ID
-
-Bugs:
-- Fixed focus problem with Samsung devices
-- Fixed problems with new google keyboard
-
-### 2.3.2
-
-Improvements:
-- Better candidate gathering over slow connections
-
-#### 2.3.1
-
-Improvements
-- Handling additional cases of packet loss during video chat
-
-Bugfixes
-- Fixed problem uploading images on Android 5.x and Samsung S5 devices
-
-#### 2.3.0
-
-Improvements
-- Better handling of connection loss during the video chat
-- Logging now enabled by default. Call IDnowSDK.disableLogging() to disable logging
-- Removed Jackson dependency
-
-Bugfixes
-- Removed app_name string from SDK
-
-#### 2.2.0
-
-Improvements
-- Dont allow screenshots during identification due to security considerations
-- Success and failure redirects now support custom fields
-- Better handling of volume controls of headset plugged in
-- During check data, half of the screen is not covered anymore
-- Support for swiss environment
-- Possibility to change the example mobile number
-- Additional customization options: color of proceed_button_text, branded dialogs, color of video_subscriber_border_color and more
-
-Bugfixes
-- Fixed wrong SSL certificate validation
-
-#### 2.1.0
-
-Improvements
-- Added echo cancellation
-
+Changelog are keeped inside the CHANGELOG.md file
```

Comment:
would be better if CHANGELOG.md is link (clickable)

Reply:
Using relative path (https://github.blog/2013-01-31-relative-links-in-markup-files/), you just need to add the following to `Readme.md` where you think it's the most suitable place :

```markdown
[Changelog](CHANGELOG.md)
```